modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -37,19 +40,23 @@
 import org.openapitools.codegen.meta.Stability;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import com.github.curiousoddman.rgxgen.RgxGen;
 
 import java.time.OffsetDateTime;
 import java.time.ZoneOffset;
 import java.time.format.DateTimeFormatter;
 import java.io.File;
 import java.util.*;
 import java.util.regex.Pattern;
+import java.util.regex.Matcher;
 
+import static org.openapitools.codegen.utils.OnceLogger.once;
 import static org.openapitools.codegen.utils.StringUtils.camelize;
 import static org.openapitools.codegen.utils.StringUtils.underscore;
 
 public class PythonClientExperimentalCodegen extends PythonClientCodegen {
     private static final Logger LOGGER = LoggerFactory.getLogger(PythonClientExperimentalCodegen.class);
+    private Map<String, Schema> modelNameToSchema = new HashMap<String, Schema>();
```

Comment:
This seems to be the same code as my PR https://github.com/OpenAPITools/openapi-generator/pull/7220. I've noticed other code generators have similar issues, that's why I put the map in DefaultCodeGenerator.  It would be good if we can either get my PR in then that can be reused for your PR, or even remove this part of the enhancement and when my PR is in, the performance optimization will be applied.

Reply:
Than you for letting me know about your PR
I will wait for your PR to be landed, and then I will use your code in this one.